DownloadableFontsKotlin/app/src/main/java/com/example/android/downloadablefonts/MainActivity.kt

```diff
@@ -20,44 +20,34 @@ import android.graphics.Typeface
 import android.os.Bundle
 import android.os.Handler
 import android.os.HandlerThread
-import android.support.design.widget.TextInputLayout
-import android.support.v4.provider.FontRequest
-import android.support.v4.provider.FontsContractCompat
-import android.support.v4.util.ArraySet
-import android.support.v7.app.AppCompatActivity
 import android.text.Editable
 import android.text.TextWatcher
 import android.util.Log
 import android.view.View
-import android.widget.ArrayAdapter
-import android.widget.AutoCompleteTextView
-import android.widget.Button
-import android.widget.CheckBox
-import android.widget.ProgressBar
-import android.widget.SeekBar
-import android.widget.TextView
-import android.widget.Toast
-
-import java.util.Arrays
-
+import android.widget.*
+import androidx.appcompat.app.AppCompatActivity
+import androidx.collection.ArraySet
+import androidx.core.provider.FontRequest
+import androidx.core.provider.FontsContractCompat
 import com.example.android.downloadablefonts.Constants.ITALIC_DEFAULT
 import com.example.android.downloadablefonts.Constants.WEIGHT_DEFAULT
 import com.example.android.downloadablefonts.Constants.WEIGHT_MAX
 import com.example.android.downloadablefonts.Constants.WIDTH_DEFAULT
 import com.example.android.downloadablefonts.Constants.WIDTH_MAX
+import com.google.android.material.textfield.TextInputLayout
 
 class MainActivity : AppCompatActivity() {
 
-    lateinit private var mHandler: Handler
+    private lateinit var mHandler: Handler
 
-    lateinit private var mDownloadableFontTextView: TextView
-    lateinit private var mWidthSeekBar: SeekBar
-    lateinit private var mWeightSeekBar: SeekBar
-    lateinit private var mItalicSeekBar: SeekBar
-    lateinit private var mBestEffort: CheckBox
-    lateinit private var mRequestDownloadButton: Button
+    private lateinit var mDownloadableFontTextView: TextView
```

Comment:
I know this is outside your intended work, but can we rename all of these variables and remove the hungarian notation?